packages/jest-create-cache-key-function/src/index.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {createHash} from 'crypto';
+import {relative} from 'path';
+import {readFileSync} from 'graceful-fs';
+import type {Config, ProjectConfig} from '@jest/types';
+
+type CacheKeyOptions = {
+  config: ProjectConfig;
+  instrument: boolean;
+};
+
+type GetCacheKeyFunction = (
+  fileData: string,
+  filePath: Config.Path,
+  configStr: string,
+  options: CacheKeyOptions,
+) => string;
+
+function getGlobalCacheKey(files: Array<string>, values: Array<string>) {
+  const dependencies = require('../package').dependencies;
```

Comment:
```suggestion
  const dependencies = require('../package.json').dependencies;
```

and move out of function to avoid reading it multiple times?